tardis/model/base.py

```diff
@@ -84,6 +84,41 @@ def r_outer(self):
         return self.geometry.r_outer.values * self.geometry_units["r_outer"]
 
 
+class Radial1DGeometry:
+    """
+    Holds information about model geometry for radial 1D models.
+
+    Parameters
+    ----------
+    radius : astropy.units.quantity.Quantity
+    velocity : astropy.units.quantity.Quantity
+    """
+
+    def __init__(self, radius, velocity):
```

Comment:
the call signature should be `def __init__(self, r_inner, r_outer, v_inner, v_outer):`